app-editors/vis/vis-0.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="Vis aims to be a modern, legacy free, simple yet efficient vim-like editor."
+HOMEPAGE="https://github.com/martanne/vis"
+SRC_URI="https://github.com/martanne/vis/archive/v${PV}.tar.gz -> vis-0.3.tar.gz"
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="-selinux tre +curses"
```

Comment:
1. Sort it,
2. I don't think we *ever* use the minus-syntax here (why would you anyway?)